src/home/fetch-github/preview-to-full-mapping.ts

```diff
@@ -1,5 +1,6 @@
 import { GitHubIssue } from "../github-types";
 
+// this is not used
```

Comment:
There are some comments that seem like action items. Why not just implement? This pull doesn't seem ready.

Reply:
it wasn't ready, my bad opened wrong type of PR